### ISSUE TITLE
No Cancel event fired

### ISSUE THREAD

**Issue by msmpro** [myflashlab/fileChooser-ANE#5]
Hi!
No "Cancel" event fired if the user don't select any picture from gallery.
Then, don't have a simple way to know when the user was canceled the selection.

Thanks in advance.

**Comment by myflashlab** [comment]
Actually you should use the Error dispatch. When you receive that event, it
means that your user is back to your application.

On Saturday, February 6, 2016, msmpro <notifications@github.com> wrote:

> Hi!
> No "Cancel" event fired if the user don't select any picture from gallery
> Then, don't have a simplest way to know when the user was canceled the
> selection
>
> Thanks in advance
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/myflashlab/fileChooser-ANE/issues/5>.
>


-- 
Sent from iPhone

**Comment by msmpro** [comment]
I'm using this code, and and Error event it's not fired when file selection is cancelled:

```
var fChooser:FileChooser = new FileChooser;
fChooser.addEventListener(FileChooserEvent.ERROR, fChooserOnError);
```

I'm missing something?

**Comment by myflashlab** [comment]
OH, you're right. That was a human mistake when the dev team had wrongly commented out the dispatch method while debugging for the last quick fix! Please update to V4.0.2 and you will have the ```onError``` dispatch on both Android and iOS.

Dear @msmpro considering that this was an embarrassing mistake from our side, we would like to offer you a cool proposal to compensate the trouble you faced with our little mistake. please email us at (help at myflashlabs . com)
